src/shared/components/about.tsx

```diff
@@ -0,0 +1,115 @@
+import { Component } from "inferno";
+import { Helmet } from "inferno-helmet";
+import { i18n } from "../i18next";
+
+const title = i18n.t("about_title");
+
+export class About extends Component<any, any> {
+  constructor(props: any, context: any) {
+    super(props, context);
+  }
+  render() {
+    return (
+      <div>
+        <Helmet title={title}>
+          <meta property={"title"} content={title} />
+        </Helmet>
+        <div class="container">
+          <h1>{i18n.t("about_title")}</h1>
+          <p>The idea to make Lemmy was a combination of factors.</p>
```

Comment:
@Nutomic You mean make this whole answer translatable?

Also the title and `about` name are `i18n` strings. I kinda like about, because it means we can add other things to this page if we want.

Reply:
Of course, not everyone speaks English.